standalone-e2e-test/pom.xml

```diff
@@ -1,94 +1,149 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.github.lammaer</groupId>
-  <artifactId>standalone-e2e-test</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  
-  <parent>
-	 <groupId>com.github.kovaku</groupId>
-	 <artifactId>userservice</artifactId>
-	 <version>1.0</version>
-  </parent>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.lammaer</groupId>
+    <artifactId>standalone-e2e-test</artifactId>
+    <version>1.0-SNAPSHOT</version>
 
-  <properties>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
-    <skipTests>true</skipTests>
-    <spring.profiles.active>dev</spring.profiles.active>
-   </properties>
+    <parent>
+        <groupId>com.github.kovaku</groupId>
+        <artifactId>userservice</artifactId>
+        <version>1.0</version>
+    </parent>
 
-  <name>standalone-e2e-test</name>
-  <url>http://maven.apache.org</url>
-
-  <dependencies>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-test</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-logging</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.rest-assured</groupId>
-      <artifactId>rest-assured</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.rest-assured</groupId>
-      <artifactId>json-schema-validator</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.rest-assured</groupId>
-      <artifactId>json-path</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.rest-assured</groupId>
-      <artifactId>xml-path</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
-
-  <profiles>
-    <profile>
-      <id>dev</id>
-      <properties>
+    <!-- default settings, used during the full build (mvn clean install) of the service
+    We run E2E tests against a freshly built and launched webservice docker image
+     -->
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
The compiler config can be outside of the profile I think, since it should be applied all cases.